azure_monitor/setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = opentelemetry-azure-monitor-exporters-python
+description = Azure Monitor exporters using OpenTelemetry
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Microsoft
+author_email = appinsightssdk@microsoft.com
```

Comment:
TelReachSDK@microsoft.com?

Reply:
That name doesn't make sense.

Reply:
Well appinsightssdk@microsoft.com is actually an alias for node.js not python, maybe updating the alias to actually include all AppInsights SDK people would be nice

Reply:
We will keep appinsightssdk@microsoft.com and update the alias to include our team.